third_party/IsAppEncrypted/IsAppEncrypted.m

```diff
@@ -18,27 +18,6 @@
 // LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 // FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
 // IN THE SOFTWARE.
-//
```

Comment:
Isn't this documentation that should stay?

Reply:
I moved it to internal target's README (see CL). I'll undo this change so this piece of documentation remains intact.

Reply:
WDYT of it now? I figured it's enough to have it only in the implementation.